### ISSUE TITLE
Responsive Bar Chart Example Is Not Actually Responsive

### ISSUE THREAD

**Issue by PBI-David** [vega/vega-lite#9226]
Using the code from the responsive bar chart example in the main editor:

```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
  "width": "container",
  "height": 250,
  "data": {"url": "data/cars.json"},
  "mark": "bar",
  "encoding": {
    "x": {"field": "Origin"},
    "y": {"aggregate": "count", "title": "Number of Cars"}
  }
}

```

I have tried in both FF and Chrome and I'm not sure it is behaving correctly. 

If  you resize the container to be more narrow, the bars do not have equal widths. In fact, one can be cut off:

![image](https://github.com/vega/vega-lite/assets/86846675/68411529-cd0e-4c98-8a51-ac31064c0bf5)

![image](https://github.com/vega/vega-lite/assets/86846675/00d8ba2b-de85-4fce-bcb3-37e93c782423)

I'm not sure if this is a regression or it has always been this way but it doesn't look right.

**Comment by domoritz** [comment]
Works for me in safari (a bit janky but works).

https://github.com/vega/vega-lite/assets/589034/882f64ac-9155-40fd-a234-f6f5891b4282

**Comment by PBI-David** [comment]
@domoritz  - your video shows it **not** working though. The bars are not proportionally sized. 

On further investigation, I can get this working in FF and Chrome if I wiggle the mouse around on resize. I think this is something to do with the mouse/point event functionality that was recently introduced in Vega. It might be an idea to move this to Vega as the raw Vega spec has the exact same issues.

**Comment by domoritz** [comment]
Oh, I see what you mean. I can wiggle a bit and then it's correct. I suspect it's some issue with the editor but would need to investigate more. 

https://github.com/vega/vega-lite/assets/589034/5779c3a0-4b39-44a1-8238-7b9bfa63ac61

**Comment by PBI-David** [comment]
Are you sure it is the editor and not related to https://github.com/vega/vega/issues/3825. 

![image](https://github.com/vega/vega-lite/assets/86846675/3b019267-2121-40e1-898e-aa3913c1b2e8)

**Comment by domoritz** [comment]
No, I am not and there could be some interaction here. But resizing should only be driven by the size of the container and not any mouse or cursor events.

**Comment by PBI-David** [comment]
Closing as this now seems to work fine.
